app/commands/thredded/notify_following_users.rb

```diff
@@ -6,13 +6,13 @@ def initialize(post)
     end
 
     def run
-      return unless targetted_users.present?
-      PostMailer.post_notification(@post.id, targetted_users.map(&:email)).deliver_now
-      MembersMarkedNotified.new(@post, targetted_users).run
+      return if targeted_users.empty?
+      PostMailer.post_notification(@post.id, targeted_users.map(&:email)).deliver_now
+      MembersMarkedNotified.new(@post, targeted_users).run
```

Comment:
@jayroh I'm wondering if we should actually be doing this. Should we perhaps send separate notifications to each person and mark them as notified. Otherwise we're divulging email addresses to all followers I think (ok in my use case, not in all). Could be wrapped up in one job/method, but per user